stable/nginx-ingress/ci/deployment-tcp-udp-configMapNamespace-values.yaml

```diff
@@ -0,0 +1,11 @@
+controller:
```

Comment:
You will need to also add the `type: ClusterIP` here.

```
  service:
    type: ClusterIP
```

Reply:
done